lms/static/js/spec/commerce/receipt_spec.js

```diff
@@ -0,0 +1,186 @@
+define([
+        'jquery', 'jquery.ajax-retry',
+        'js/commerce/views/receipt_view', 'common/js/spec_helpers/ajax_helpers'
+    ],
+    function ($, AjaxRetry, ReceiptView, AjaxHelpers){
+        'use strict';
+        describe('edx.commerce.ReceiptView', function() {
+            var data, courseResponseData, providerResponseData = null;
+
+            var createReceiptView = function() {
+                return new ReceiptView({el: $('#receipt-container')});
+            };
+
+            var loadReceiptFixture = function() {
+                var receiptFixture = readFixtures('templates/commerce/receipt.underscore');
+                var providerFixture = readFixtures('templates/commerce/provider.underscore');
+                appendSetFixtures(
+                    '<script id=\"receipt-tpl\" type=\"text/template\" >' + receiptFixture + '</script>'
+                );
+                appendSetFixtures(
+                    '<script id=\"provider-tpl\" type=\"text/template\" >' + providerFixture + '</script>'
+                );
```

Comment:
Can you not just pass both script tags into 1 ```appendSetFIxtures``` function, instead of calling the function twice.